src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+@import url(
+  "https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+);
+
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo_link {
+  line-height: 0;
+  margin: 10px 0 10px 50px;
+  cursor: pointer;
+}
+
+.nav_link {
+  text-decoration: none;
+  color: #000;
+  padding: 0 10px;
```

Comment:
you don't need horiz padding, just add right or left margin (not for the last child)